source/pervasive/std_specs/alloc.rs

```diff
@@ -1,11 +1,221 @@
 use crate::prelude::*;
+use builtin::*;
 extern crate alloc;
 
+use alloc::vec::Vec;
+use core::option::Option;
+use core::option::Option::Some;
+use core::option::Option::None;
+use core::alloc::Allocator;
+
 verus!{
 
-// TODO need to figure out how to handle the Allocator type param
-//#[verifier(external_type_specification)]
-//#[verifier(external_body)]
-//pub struct ExVec<V>(alloc::vec::Vec<V>);
+#[verifier(external_type_specification)]
+#[verifier(external_body)]
+#[verifier::accept_recursive_types(T)]
+#[verifier::reject_recursive_types(A)]
+pub struct ExVec<T, A: Allocator>(Vec<T, A>);
```

Comment:
Should the `Vec` declarations go in `std_specs/vec.rs`?  They might be hard for people to find in `alloc.rs`.